Makefile

```
@@ -100,7 +100,7 @@ test.mixed:
 		$(REGRESSION_TEST_MODE)
 	@ echo "herd7 AArch64 mixed instructions tests: OK"
 
-test::
+test.neon::
```

Why not perform neon tests by default?
```suggestion
test:: test.neon
test.neon::
```